.gitignore

```diff
@@ -8,4 +8,7 @@ build/
 node_modules
 
 # npm logs
-npm-debug.log*
\ No newline at end of file
+npm-debug.log*
+
+# rollup-plugins-typescript2
+.rpt2_cache
```

Comment:
I'll take a look although on quick inspection it showed up as CR LF in my text editor.